spyder/plugins/toolbar/plugin.py

```diff
@@ -43,11 +44,13 @@ class Toolbar(SpyderPluginV2):
     def get_name():
         return _('Toolbar')
 
-    def get_description(self):
+    @staticmethod
+    def get_description():
         return _('Application toolbars management.')
```

Comment:
```suggestion
        return _('Manage application toolbars.')
```

Reframe this like the others